gamemodes/horde/entities/entities/horde_watchtower/init.lua

```diff
@@ -83,13 +84,27 @@ hook.Add("PlayerUse", "PickUpWatchtower", function(ply, ent)
                 end
             end
         end
+        if not ply.IsHoldingObject then
         local p = ent:GetPos()
```

Comment:
this should be indented

Reply:
wasn't sure if you were referring to the line itself or the block of code it surrounds so I indented the block